### PR TITLE
Adding to stix dialect feature

### PR DESCRIPTION
**Pull request by mdazam1942**

### CODE REVIEW

File: stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py [hunk 1]
```diff
@@ -461,6 +456,7 @@ def test_ocsf_translation_prop(self):
         """
         result_file = open('stix_shifter_modules/aws_athena/tests/stix_translation/json/ocsf_results.json', 'r').read()
         data = json.loads(result_file)
+        map_data = entry_point.get_results_translator('ocsf').map_data
```
Comment (mdazam1942) [inline stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py:459]: this unittest fails.

File: stix_shifter_utils/utils/file_helper.py [hunk 2]
```diff
@@ -9,15 +9,18 @@
...
+def read_json(filepath, options, search_path=__default_search_path, key_subsitute=None):
     # Read JSON file that is either passed in with the options or internally contained in the module
     # logger.debug('call: read_json: ' + json.dumps(options, indent=4))
     # filepath may be:
     #  'to_stix_map.json' -> 'to_stix_map' mapping data if present otherwise contents of 'module'/stix_translation/json/to_stix_map.json
     #  'to_stix_map' -> 'to_stix_map' mapping data if present otherwise contents of 'module'/stix_translation/json/to_stix_map.json
     #  '/full/path/somefile.json' -> 'somefile' mapping data if present otherwise contents of /full/path/somefile.json
     file_name = Path(filepath).name
-    file_key = file_name
+    if key_subsitute:
```
Comment (mdazam1942) [inline stix_shifter_utils/utils/file_helper.py:20]: not clear what `key_subsitute` is doing.

File: stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py (hunk 1 above)
Comment (baulus) [reply stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py:459]: works now

File: stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py [hunk 3]
```diff
@@ -151,27 +149,27 @@ def test_common_prop(self):
...
+            "starttime": 1592547796,
+            "endtime": 1592547798,
+            "action": "REJECT",
+            "date": "2020-06-19",
+            "logstatus": "OK",
+            "numbytes": 40,
+            "region": "us-east-1",
+            "version": 2
         }
+        map_data = entry_point.get_results_translator('vpcflow').map_data
```
[the remark this answers is not in the document]
Comment (baulus) [reply stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py:168]: my last comment is outdated

**Review by baulus — APPROVED** [review]

**Review by delliott90 — APPROVED** [review]

**Pull request merged**